service/qos/service_level_controller.cc

```diff
@@ -27,6 +27,7 @@
 #include "db/system_distributed_keyspace.hh"
 
 namespace qos {
+static logging::logger sl_logger("workload prioritization");
```

Comment:
Is "workload prioritization" the right name for this? In other places you called it "service level".

Reply:
Right, "workload prioritization" bears no meaning in open-source scylla, so "service level" would be a better fit, especially that it's likely that more parameters would be added to service levels later.